src/display_buffer.cc

```diff
@@ -312,7 +317,16 @@ DisplayLine parse_display_line(StringView line)
                 auto closing = std::find(it+1, end, '}');
                 if (closing == end)
                     throw runtime_error("unclosed face definition");
-                face = get_face({it+1, closing});
+                if (*(it+1) == '`')
```

Comment:
In the javascript world, templates often use the [mustache notation](https://github.com/wycats/handlebars.js), so you're not too far off with the current markup.